challenges/objects-arrays.js

```diff
@@ -59,11 +89,17 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i = 0; i < graduates.length; i++) {
+    contactInfo.push(graduates[i].first_name + ' ' + graduates[i].email);
+}
+
```

Comment:
this works, or you could do it with an array method.  

const contactInfo = graduates.map( graduate => graduate.first_name + ' ' + graduate.email);